python/feature_tokenization.py

```diff
@@ -8,9 +8,9 @@
 
 from sklearn.feature_extraction.text import CountVectorizer
 
-# Setting top-level parameters
+# %% Setting top-level parameters
 MIN_DF = 5
-NO_VITALS = True
```

Comment:
@scotthlee I also experimented with just dumping everything in (including vitals). We can revert before final depending on what we'd like